dastardcommander/demo/scrollbar/demo.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Can you add that `QScrollArea` doesn't appear in the code, so this example can only be understood by inspection in qtdesigner?

Reply:
Frankly, I doubt a demo that relies on usage of a .ui file will be very useful. In the actual dcom code you use a `QScrollArea` from code, so this demo doesn't even help me understand how that works aside from the comments.

Reply:
Okay, noted. I am still holding onto this demo, because it was hard-won knowledge. I added comments to that effect.